src/scenes/Event/components/EventInfo/styles.js

```diff
@@ -10,7 +10,8 @@ const styles = (theme) => ({
     textOverflow: 'ellipsis',
     whiteSpace: 'nowrap',
     overflow: 'hidden',
-    paddingLeft: theme.padding.lg.px,
+    paddingLeft: theme.padding.md.px,
+    fontSize: theme.typography.fontSize,
```

Comment:
why is this necessary? theme.typography.fontSize is just the default font size right?

Reply:
the Result Setter text is being ellipsized. i changed the font size because i wanted it black color still, just smaller text size. suggestions?

Reply:
oh i did that ellipsis thing on purpose but now as i think of it, it doesn't make much sense. Can you remove textOverflow: 'ellipsis', whiteSpace: 'nowrap',overflow: 'hidden',?

Reply:
done. are you ok with the text size change or you think it should be big?

Reply:
I'm ok with the smaller font size since it would be nice to make the address in one line on most of the laptops